examples/mnist/train_mnist_data_parallel_updater.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import argparse
+
+import chainer
+import chainer.links as L
+from chainer import training
+from chainer.training import extensions
+import chainerx
+import sys
+
+import train_mnist
+
+
+def main():
+    # This script is almost identical to train_mnist.py. The only difference is
+    # that this script uses data-parallel computation on two GPUs.
+    # See train_mnist.py for more details.
+    parser = argparse.ArgumentParser(description='Chainer example: MNIST')
+    parser.add_argument('--batchsize', '-b', type=int, default=400,
+                        help='Number of images in each mini-batch')
+    parser.add_argument('--epoch', '-e', type=int, default=20,
+                        help='Number of sweeps over the dataset to train')
+    parser.add_argument('--out', '-o', default='result_data_parallel',
+                        help='Directory to output the result')
+    parser.add_argument('--resume', '-r', default='',
+                        help='Resume the training from snapshot')
+    parser.add_argument('--unit', '-u', type=int, default=1000,
+                        help='Number of units')
+    parser.add_argument('--devices', '-d', type=str, nargs='*',
+                        default=['0', '1', '2', '3'],
+                        help='Device specifiers. Either ChainerX device '
+                        'specifiers or integers. If non-negative integer, '
+                        'CuPy arrays with specified device id are used. If '
+                        'negative integer, NumPy arrays are used')
+    parser.add_argument('--ljob', '-j', type=int,
```

Comment:
Better to set a default value.

```suggestion
    parser.add_argument('--ljob', '-j', type=int, default=4,
```